src/core/api/public_api.ts

```diff
@@ -611,7 +622,58 @@ class Player extends EventEmitter<IPublicAPIEvent> {
   loadVideo(opts : ILoadVideoOptions) : void {
     const options = parseLoadVideoOptions(opts);
     log.info("API: Calling loadvideo", options);
+    this._priv_lastContentPlaybackInfos = { options };
+    this._priv_initializeContentPlayback(options);
+  }
 
+  /**
+   * Reload last content. Init media playback without fetching again
+   * the manifest.
+   * @param {Object} reloadOpts
+   */
+  reload(reloadOpts?: { startAt?: { position?: number; relative?: number } }): void {
+    const { options,
+            manifest,
+            lastPlaybackPosition } = this._priv_lastContentPlaybackInfos;
+    if (options === undefined ||
+        manifest === undefined ||
+        lastPlaybackPosition === undefined) {
+      throw new Error("API: Can't reload without having previously loaded a content.");
+    }
+    let startAtPositon: number;
+    if (reloadOpts !== undefined &&
+        reloadOpts.startAt !== undefined &&
+        reloadOpts.startAt.position !== undefined) {
+      startAtPositon = reloadOpts.startAt.position;
+    } else {
+      let playbackPosition: number;
+      if (this.state === "STOPPED" || this.state === "ENDED") {
+        playbackPosition = lastPlaybackPosition;
+      } else {
+        if (this.videoElement === null) {
+          throw new Error("API: Can't reload when video element does not exist.");
+        }
+        playbackPosition = this.videoElement.currentTime;
+      }
+      if (reloadOpts !== undefined &&
+          reloadOpts.startAt !== undefined &&
+          reloadOpts.startAt.relative !== undefined) {
```

Comment:
Also here, difference with `reloadOpts?.startAt?.relative !== undefined`?

Reply:
Typescript doesn't understand that if I use optional chaining here, the `relative` property will be defined when I use it ...

Reply:
Maybe as:
```ts
const relative = reloadOpts?.startAt?.relative;
if (relative !== undefined) {
```